Libraries/src/Amazon.Lambda.MQEvents/ActiveMQEvent.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace Amazon.Lambda.MQEvents
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Apache ActiveMQ event
+    /// https://docs.aws.amazon.com/lambda/latest/dg/with-mq.html
+    /// </summary>
+    public class ActiveMQEvent
+    {
+        /// <summary>
+        /// The source of the event.
+        /// </summary>
+        public string EventSource { get; set; }
+
+        /// <summary>
+        /// The ARN of the event source.
+        /// </summary>
+        public string EventSourceArn { get; set; }
+
+        /// <summary>
+        /// List of ActiveMQ messages.
+        /// </summary>
+        public IList<ActiveMQMessage> Messages { get; set; }
+
+        /// <summary>
+        /// Class that represents an ActiveMQ message..
+        /// </summary>
+        public class ActiveMQMessage
+        {
+            /// <summary>
+            /// Message ID
+            /// </summary>
+            public string MessageId { get; set; }
+
+            /// <summary>
+            /// Message Type
+            /// </summary>
+            public string MessageType { get; set; }
+
+            /// <summary>
+            ///  Data sent to Active MQ broker encoded in Base 64.
+            /// </summary>
+            public string Data { get; set; }
+
+            /// <summary>
+            /// Connection ID
+            /// </summary>
+            public string ConnectionId { get; set; }
+
+            /// <summary>
+            /// Indicates if the message was redelivered.
+            /// </summary>
+            public bool? Redelivered { get; set; }
+
+            /// <summary>
+            /// Indicates if the message is persistent.
+            /// </summary>
+            public bool? Persistent { get; set; }
+
+            /// <summary>
+            /// Message Destination
+            /// </summary>
+            public Destination Destination { get; set; }
+
+            /// <summary>
+            /// Message Timestamp
+            /// </summary>
+            public long? Timestamp { get; set; }
```

Comment:
I haven't worked with our event packages a lot - do we ever convert these to `DateTime` here? Or does the user have to do that?

Reply:
Ideally that should be the case. However, I have seen instances where converting epoch time to DateTime gives an error in our Lambda library (there is already an open issue https://github.com/aws/aws-lambda-dotnet/issues/839). Also, this was ported from https://github.com/aws/aws-lambda-java-libs/blob/master/aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/ActiveMQEvent.java, which uses `long`. So wanted to be consistent here.

Reply:
Sounds good, thanks for the context.